Makefile

```diff
@@ -10,7 +10,7 @@ HANDLER_IMAGE ?= $(IMAGE_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)
 
 unit_test_args ?=  -r --randomizeAllSpecs --randomizeSuites --race --trace $(UNIT_TEST_ARGS)
 
-export KUBEVIRT_PROVIDER ?= k8s-1.15.1
+export KUBEVIRT_PROVIDER ?= k8s-1.16.2
```

Comment:
Please bump the version also in https://github.com/nmstate/kubernetes-nmstate/blob/master/docs/developer-guide-commands.md#local-cluster. It is there on two places.